chapter_linear-regression/linear-regression-scratch.md

```diff
@@ -89,6 +99,19 @@ class LinearRegressionScratch(d2l.Module):  #@save
             self.b = tf.Variable(b, trainable=True)
 ```
 
+```{.python .input}
+%%tab jax
+class LinearRegressionScratch(d2l.Module):  # @save
+    num_inputs: int
+    lr: float
+    sigma: float = 0.01
+
+    def setup(self):
+        self.w = self.param('w', nn.initializers.normal(self.sigma),
+                            (self.num_inputs, 1))
+        self.b = self.param('b', nn.initializers.zeros, (1))
+```
```

Comment:
merge this into the above code block by using `if tab.selected('jax'):`. Otherwise there's no point to only merge the other three framework implementations and keep jax separate